docs/design/3_2_multirotor_v2.tex

```diff
@@ -1,26 +1,250 @@
+	\includegraphics[width=0.85\textwidth]{img/multirotor.png}
+	\caption[Team 109 Multirotor Drone]{Team 109 Multirotor Drone}
+	\label{fig:drone1}
+\end{figure}
+
+A remote-piloted aerial system (RPAS) or sometimes referred to as ``drone" and ``multirotor" consists of serveral subsystems: the airframe, mechanical structures, propulsion, battery, sensors, and flight computer. This section will explore these subsystems in more detail. Figure~\ref{fig:drone1} is a render of the completed multirotor with the computing platform payload attached.
+
+\subsubsection{Mechanical}\label{section:drone-mech}
+
+The frame and mounts which holds the propulsion system, on-board computer/sensors, and the payload together makes up most of the mechanical subsystem of the multirotor.
 
-\subsubsection{Mechanical}
 \paragraph{Frame Configuration}
+
+\begin{figure}[b]
+	\centering
+	\begin{subfigure}[b]{0.3\textwidth}
+		\centering
+		\includegraphics[scale=0.4]{img/drone_yconfig}
+		\caption{Tricopter Y-Configuration}
+		\label{fig:tricopter-y}
+	\end{subfigure}
+	~
+	\begin{subfigure}[b]{0.3\textwidth}
+		\centering
+		\includegraphics[scale=0.4]{img/drone_xconfig}
+		\caption{Quadcopter X-Configuration}
+		\label{fig:quadcopter-x}
+	\end{subfigure}
+	~
+	\begin{subfigure}[b]{0.3\textwidth}
+		\centering
+		\includegraphics[scale=0.4]{img/drone_hexconfig}
+		\caption{Hexcopter X-Configuration}
+		\label{fig:hexcopter-x}
+	\end{subfigure}
+	
+	\caption{Multirotor RPAS Configurations}
+	\label{fig:rpas-config}
+\end{figure}
+
+\begin{figure}[h]
+	\centering
+	\includegraphics[scale=0.4]{img/drone_xconfigt}
+	\caption{Quadcopter X-configuration}
+	\label{fig:quadcopter-x-t}
+\end{figure}
+
+\begin{figure}[h]
+	\centering
+	\begin{subfigure}[b]{0.3\textwidth}
+		\centering
+		\includegraphics[scale=0.4]{img/drone_x_pitch}
+		\caption{Pitch forward}
+		\label{fig:x-pitch}
+	\end{subfigure}
+	~
+	\begin{subfigure}[b]{0.3\textwidth}
+		\centering
+		\includegraphics[scale=0.4]{img/drone_x_roll}
+		\caption{Roll left}
+		\label{fig:x-roll}
+	\end{subfigure}
+	~
+	\begin{subfigure}[b]{0.3\textwidth}
+		\centering
+		\includegraphics[scale=0.4]{img/drone_x_yaw}
+		\caption{Yaw left}
+		\label{fig:x-yaw}
+	\end{subfigure}
+	
+	\caption{Using differential thrust to achive 6-DOF movement. }
+	\label{fig:rpas_6dof}
+\end{figure}
+
+Multirotors have different configurations such as tricopter, quadcopter, hexacopter as depicted in Figure~\ref{fig:rpas-config}. As each frame has different motor layouts, the forces acting on the multirotor are inherently different and thus each configuration provides particular advantages and disadvantages.
+
+We use a quadcopter X4 configuration (4 motors and propellers in the shape of a letter X) for our multirotor because it is easy to control and provides abundant control system robustness. Such configuration is also the most popular amongst the UAS industry because for its balance between redundancy and robustness with capacity and performance.
```

Comment:
```suggestion
We use a quadcopter X4 configuration (4 motors and propellers in the shape of a letter X) for our multirotor because it is easy to control and provides abundant control system robustness. Such configuration is also the most popular amongst the UAS industry for its balance between redundancy and robustness with capacity and performance.
```